src/Revenue/Listeners/UpdateRevenueWhenDonationAmountUpdated.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Give\Revenue\Listeners;
+
+use Give\Donations\Models\Donation;
+use Give\Revenue\Repositories\Revenue;
+
+/**
+ * @unreleased
+ */
+class UpdateRevenueWhenDonationAmountUpdated
+{
+    /**
+     * @unreleased
+     *
+     * @param int $paymentID The ID of the payment.
+     */
+    public function __invoke($paymentID)
+    {
+        give(Revenue::class)->updateRevenueAmount(
+            Donation::find($paymentID)
```

Comment:
@kjohnson we want to avoid using the legacy term `paymentId` for new code written about donations (even though the do_action uses it).  Instead let's make sure to use `donationId`

```suggestion
     * @param int $donationId The ID of the Donation.
     */
    public function __invoke($donationId)
    {
        give(Revenue::class)->updateRevenueAmount(
            Donation::find($donationId)
```